src/reconstruction/enzyme_constrained.jl

```diff
@@ -29,20 +29,35 @@ ecmodel = make_enzyme_constrained_model(
         "total" => 0.5,
     ),
 )
-```
 
+ecmodel2 = make_enzyme_constrained_model(
+    model;
+    total_gene_product_mass_bound = 0.5
+)
+```
 # Notes
 Reactions with no turnover number data, or non-enzymatic reactions that should
-be ignored, must have `nothing` in the `gene_associations` field of the
+be ignored, *must* have `nothing` in the `gene_associations` field of the
 associated reaction.
+
+The capacity bound
```

Comment:
?

Reply:
oh no! something got lost with all the rebasing :/